src/NUnitEngine/nunit.engine.tests/Services/TestRunnerFactoryTests/Results/Net20SingleProjectExpectedRunnerResults.cs

```diff
@@ -0,0 +1,236 @@
+﻿// ***********************************************************************
+// Copyright (c) 2019 Charlie Poole, Rob Prouse
+//
+// Permission is hereby granted, free of charge, to any person obtaining
+// a copy of this software and associated documentation files (the
+// "Software"), to deal in the Software without restriction, including
+// without limitation the rights to use, copy, modify, merge, publish,
+// distribute, sublicense, and/or sell copies of the Software, and to
+// permit persons to whom the Software is furnished to do so, subject to
+// the following conditions:
+//
+// The above copyright notice and this permission notice shall be
+// included in all copies or substantial portions of the Software.
+//
+// THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND,
+// EXPRESS OR IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF
+// MERCHANTABILITY, FITNESS FOR A PARTICULAR PURPOSE AND
+// NONINFRINGEMENT. IN NO EVENT SHALL THE AUTHORS OR COPYRIGHT HOLDERS BE
+// LIABLE FOR ANY CLAIM, DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION
+// OF CONTRACT, TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION
+// WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
+// ***********************************************************************
+
+using System;
+using NUnit.Engine.Internal;
+using NUnit.Engine.Runners;
+
+namespace NUnit.Engine.Tests.Services.TestRunnerFactoryTests.Results
+{
+#if !NETCOREAPP
+    internal static class Net20SingleProjectExpectedRunnerResults
+    {
+        public static RunnerResult ResultFor(ProcessModel processModel, DomainUsage domainUsage)
+        {
+            switch (processModel)
+            {
+                case ProcessModel.Default:
+                    return GetProcessModelDefaultResult(domainUsage);
+                case ProcessModel.InProcess:
+                    return GetProcessModelInProcessResult(domainUsage);
+                case ProcessModel.Separate:
+                    return GetProcessModelSeparateResult(domainUsage);
+                case ProcessModel.Multiple:
+                    return GetProcessModelMultipleResult(domainUsage);
+                default:
+                    ThrowOutOfRange(processModel);
+                    break;
+            }
+
+            throw new ArgumentOutOfRangeException(
+                $"No expected Test result provided for this {nameof(ProcessModel)}/{nameof(DomainUsage)} combination.");
+        }
+
+        private static RunnerResult GetProcessModelMultipleResult(DomainUsage domainUsage)
+        {
+            switch (domainUsage)
+            {
+                case DomainUsage.Default:
+                    return new RunnerResult
+                    {
+                        TestRunner = typeof(MultipleTestProcessRunner),
+                        SubRunners = new[]
+                        {
+                            new RunnerResult {TestRunner = typeof(ProcessRunner)}
```

Comment:
ReSharper overrides Visual Studio's default and removes the spaces from inside `{` and `}`. If you want to worry about it, we could add this to the .editorconfig to go back to the more common style:

```ini
resharper_space_within_single_line_array_initializer_braces = true
```

Reply:
I'm not too worried with these - they're inconsistent across the codebase. Would you prefer we went one way or the other? Which way? 🙂

Reply:
I do have a preference for the editorconfig line, but I don't like to impose my preference unless y'all have the same preference. Just offering info on ReSharper's behavior in case it's useful.

Reply:
I _think_ I've added a change to the Resharper settings to solve that. 😄